mysql-test/suite/sys_vars/t/innodb_buffer_pool_load_now_basic.test

```diff
@@ -5,6 +5,19 @@
 -- source include/have_innodb.inc
 --source include/not_embedded.inc
 
+# Restart mysqld to ensure that the status variables below do not contain
+# stale value from previous executions of this or other tests. The following
+# race could happen without this restart:
+# 0. innodb_buffer_pool_load_status is 'Buffer pool(s) load completed ...'
+# 1. SET innodb_buffer_pool_load_now = ON
+# (1. does not start executing yet)
+# 2. Wait for innodb_buffer_pool_load_status to become '... completed ...',
+#    the condition is satisfied immediately
```

Comment:
Ah, OK - then LGTM. Is there a way of knowing which bug fixes are backported, so maybe the code review process could be quicker ?